reana_server/rest/launch.py

```diff
@@ -186,6 +163,9 @@ def launch(user, url, name="", parameters="{}", spec=None):
     except HTTPError as e:
         logging.error(traceback.format_exc())
         return jsonify(e.response.json()), e.response.status_code
+    except json.JSONDecodeError:
+        logging.error(traceback.format_exc())
+        return jsonify({"message": "Parameters contains not valid JSON."}), 400
```

Comment:
Minor: what do you think about using `The 'parameters' field is not valid JSON`? This makes it clear that `parameters` refers to that specific field and not to the parameters of `/launch` in general

Reply:
This naming problem kept me reflecting from time to time when developing the UI part. What I've done in the end was call them "workflow parameters" to differentiate them from the query string parameters. @mdonadoni suggestion is very valid and can be completed by prepending "workflow" before `'parameters'`: `The workflow 'parameters' field is not valid JSON`. Let me know what you think.

Reply:
Thank you both for a good remark, fixed it accordingly